src/Emit.hs

```diff
@@ -739,9 +739,10 @@ paramListToC xobjs = intercalate ", " (map getParam xobjs)
         getParam invalid = error (show (InvalidParameter invalid))
 
 projectIncludesToC :: Project -> String
-projectIncludesToC proj = intercalate "\n" (map includerToC (projectIncludes proj)) ++ "\n\n"
+projectIncludesToC proj = intercalate "\n" (map includerToC includes) ++ "\n\n"
   where includerToC (SystemInclude file) = "#include <" ++ file ++ ">"
         includerToC (RelativeInclude file) = "#include \"" ++ file ++ "\""
+        includes = reverse $ projectIncludes proj
```

Comment:
This is merged now in the other PR, right?

Reply:
Yes. I can rebase on current master if you want.

Reply:
Yes please!

Reply:
Done.